docs/quick-answers/linode-platform/cloud-manager-search/index.md

```diff
@@ -13,80 +13,98 @@ contributor:
 
-    type:linode AND tag:production
+### OR
 
-## Negate Operators
+`OR` can be used to search for multiple different keywords in a single query. For example, use the following query to display all Linodes and Images on your account:
 
-A negate operator can be used to return all values that are **NOT** defined by the negate operator. This will always be preceded by the `-` symbol. For example, see the following query:
+    type:linode OR type:image
 
-    -type:linode
+### AND
 
-When used, you will see all Linodes excluded from your query.
+`AND` will allow you to narrow your search more precisely for additional qualifiers. The following search finds all of th Linodes which also match the `production` tag:
 
-## Nested Queries
+    type:linode AND tag:production
 
-Nested Queries can also be performed for an even higher level of control. If you want to search for all of your domains, as well as all of your Linodes that match the tag `my-tag`for example, you can use the following Nested Query:
+### NOT
 
-    type:domain OR (type:linode AND tag:my-tag)
+The **NOT**, or negate, operator is used by preceding the keyword with a `-` symbol. A negate operator can be used to return all values that are **NOT** defined by the negate operator. For example, this query will return everything except Linodes:
+
+    -type:linode
 
 ## Aliases
 
-In addition most operators include a number of aliases that you can use as an alternative to standard usage. All aliases and their standard equivalent can be seen in the chart below.
+Some operators and keywords include one or more aliases that you can use as an alternative to standard usage. Aliases and their standard equivalent(s) are in the chart below.
 
-| Standard | Aliases |
+| Standard | Alias(es) |
 |:--------------|:------------|
 | **AND** | **&&** |
-|   **OR**  |  **&#124;****&#124;** |
-|  **tags**   | **tag** and **group** |
-|   **label** | **name** and **title** |
+| **OR** |  **&#124;****&#124;** |
+| **tag** | **tags** or **group** |
+| **label** | **name** or **title** |
+| **type** | **is** |
+
+## Nested Queries
 
-## Combining it All
+You can nest queries using parentheses for an even higher level of control. For example, if you want to search for all of your domains created in 2010, as well as all of your Linodes that match the tag `my-tag`, you can use the following nested query:
 
-The more robust functionality of the search tool can be used in combination with other features. For example, you can see all of the advanced features outlined in this guide in a single query:
+    (type:domain AND created:2010) OR (type:linode AND tag:my-tag)
+
+## Combining Queries
+
+The most robust functionality of the search tool can be used by combining and grouping queries to create compound queries. For example, you can see all of the advanced features outlined in this guide in a single query:
 
     type:linode AND -group:production || (type:NodeBalancer && tag:mytag)
 
+## Invalid Search Results
```

Comment:
Added this section - just thought it would be helpful to show what happens if there are no matches or something went wrong.